code/modules/reagents/recipes/recipes_slime.dm

```diff
@@ -146,15 +146,11 @@
 	var/obj/item/stack/material/gold/G = new /obj/item/stack/material/gold
 	G.amount = 15
 	G.loc = get_turf(holder.my_atom)
-	var/obj/item/stack/material/platinum/P = new /obj/item/stack/material/platinum
-	P.amount = 5
-	P.loc = get_turf(holder.my_atom)
```

Comment:
Why remove platinum from the reaction?

Reply:
Sieg said its a lategame gateway material. Ideally I was going to move it to a different slime. For now I'm thinking of moving it to pink and pushing pinks potions to light pink so you don't feel like that entire line is just slime friend.